tests/skrobot_tests/test_model.py

```diff
@@ -1,10 +1,12 @@
 import unittest
 
+import copy
```

Comment:
> Imports should be grouped in the following order:
> standard library imports
> related third party imports
> local application/library specific imports

https://www.python.org/dev/peps/pep-0008/#imports
https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement